routers/routes/routes.go

```diff
@@ -148,22 +148,13 @@ func NewMacaron() *macaron.Macaron {
 			ExpiresAfter: setting.StaticCacheTime,
 		},
 	))
-	m.Use(public.StaticHandler(
-		setting.AvatarUploadPath,
-		&public.Options{
+	m.Use(public.AvatarHandler(
```

Comment:
Why merge these two handlers?

Reply:
Again this came from @masudur-rahman, looking through the code:

```
// AvatarHandler implements the macaron handler for serving user avatars and repo-avatars.
func AvatarHandler(opts *AvatarOptions) macaron.Handler {
```

So both are handled by `AvatarHandler`.